app/pages/preview/ui.test.js

```diff
@@ -545,3 +542,68 @@ test('when existing marketing data - The hosting type on premise section should
     .expect(requiresHSCNRowItem.trim()).eql('This Solution requires a HSCN/N3 connection')
     .expect(requiresHSCNRow.find('div[data-test-id="view-section-table-row-vertical"]').exists).ok();
 });
+
+test('when no existing marketing data - The contact-details section should not be rendered', async (t) => {
+  await pageSetup(t);
+
+  const contactDetailsSection = Selector('[data-test-id="view-solution-contact-details"]');
+
+  await t
+    .expect(contactDetailsSection.exists).notOk();
+});
+
+test('when existing marketing data - The contact-details section should be rendered', async (t) => {
```

Comment:
this description doesn't match the test? not sure which one is the right one 😄

Reply:
it does 😸 what do you mean? The test is to check the contact-details section is rendered.

Reply:
oh yeah I read the test description above oops!